sdcm/db_stats.py

```diff
@@ -463,7 +463,7 @@ def get_test_details(self):
         from sdcm.cluster import Setup
 
         test_details = {}
-        test_details['sct_git_commit'] = subprocess.check_output(['git', 'rev-parse', 'HEAD']).strip()
+        test_details['sct_git_commit'] = subprocess.check_output(['git', 'rev-parse', 'HEAD']).strip().decode('utf8')
```

Comment:
I think you can use, 
`text=True`
instead of decoding it 
see: https://docs.python.org/3/library/subprocess.html#subprocess.run

Reply:
thanks, fixed